MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,254 @@
 import UIKit
 
 final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
+    // MARK: - Global variables
+    private var currentQuestionIndex: Int8 = 0
+    private var correctAnswers = 0
+    private let questionsAmount = 10
+    
+    // MARK: - Another classes or structs Instances
+    private var questionsFactory: QuestionFactoryProtocol?
+    private var currentQuestion: QuizQuestion?
+    private var alertPresenter: AlertPresenterProtocol?
+    private var statisticService: StatisticService?
+    internal override var preferredStatusBarStyle: UIStatusBarStyle {
+            return .lightContent
+        }
```

Comment:
internal и return в этом случае можно опустить. 
```suggestion
    override var preferredStatusBarStyle: UIStatusBarStyle { .lightContent }
```
если нравится ваш вариант, то хотя бы поправить форматирование `control+i`